content/api/commands/session.md

```diff
@@ -0,0 +1,451 @@
+---
+title: session
+---
+
+Cache and restore [cookies](/api/cypress-api/cookies),
+[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+and
+[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+in order to reduce test setup times.
+
+<Alert type="warning">
+<p>The <code>cy.session</code> API is currently experimental, and can be enabled by
```

Comment:
If you want to stay consistent, change `cy.session` to `cy.session()`

Reply:
Fixed in d474bf0